django/contrib/auth/hashers.py

```diff
@@ -322,53 +323,76 @@ def encode(self, password, salt):
             parallelism=self.parallelism,
             hash_len=argon2.DEFAULT_HASH_LENGTH,
             type=argon2.low_level.Type.I,
+            version=self.version,
```

Comment:
Is there any good reason to specify the version? Ie I would assume we would always want the newest?